mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionsProvider.scala

```diff
@@ -318,7 +322,7 @@ class CompletionsProvider(
         mkItem(label, text.getOrElse(label))
       case CompletionValue.Document(label, doc, desc) =>
         mkItem(label, doc, filterText = Some(desc))
-      case _ =>
+      case _ => // TODO check if CompletionValue.FileSystemMember should be covered separately
```

Comment:
```suggestion
      case _ =>
```
I think it should be fine as long as it works.